continuous_integration/azure/install.bat

```diff
@@ -34,7 +34,8 @@ if %USE_CONDA% == yes (
 )
 
 :: Install spyder-kernels from master
-pip install -q --no-deps git+https://github.com/spyder-ide/spyder-kernels
+:: pip install -q --no-deps git+https://github.com/spyder-ide/spyder-kernels
+pip install -q --no-deps git+https://github.com/spyder-ide/spyder-kernels.git@refs/pull/181/head
```

Comment:
>I think you are using the latest master for spyder but not for spyder-kernels.

@impact27 I am using the latest master (which is part of that PR)

Reply:
I meant for spyder-kernels, you need commit 2bcda3b35638ed2d5611a0f6192a482ad3d16f92. But I see that you have rebased PR on the current master so `test_runcell_pdb` should work now.

Reply:
But you probably need to retrigger the tests

Reply:
Thanks :-)